src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -155,6 +186,7 @@ create_reference_table(PG_FUNCTION_ARGS)
 	Oid relationId = PG_GETARG_OID(0);
 	int shardCount = 1;
 	AttrNumber firstColumnAttrNumber = 1;
+	char *colocateWithTableName = "default";
```

Comment:
I didn't get this "Therefore we will create reference tables in wrong colocation group." Maybe an example can help.

Besides, @onderkalaci is rewriting the reference tables (#969), and when he is finished, references tables will not use this API.